src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -18,7 +18,7 @@ namespace System.Net.Tests
     public class WebClientTest
     {
         [Fact]
-        public static void DefaultCtor_PropertiesReturnExpectedValues()
+        public static void WebClient_DefaultCtor_PropertiesReturnExpectedValues()
```

Comment:
Why did you change the test names to prefix them with the classname.  We don't do that on any of our other test classes/methods.

 The class is the module under test, i.e. "WebClient".  And the first part of the test name is the method or property under test.  So, that would be Ctor or Properties or DownloadData etc.